client/FxAccountClient.js

```diff
@@ -1031,4 +1099,30 @@ define([
   return FxAccountClient;
 });
 
+var METRICS_CONTEXT_OPTIONS = {
+  context: true,
+  entrypoint: true,
+  migration: true,
+  service: true,
+  utmCampaign: true,
+  utmContent: true,
+  utmMedium: true,
+  utmSource: true,
+  utmTerm: true
+};
+
+function marshallMetricsContext (data) {
+  var metricsContext = {
+    flowId: data.flowId,
+    flowBeginTime: data.flowBeginTime
```

Comment:
Hmm, I don't *think* so. The auth server validates its payload so doing it here too feels redundant and would open up the can of worms of having the same thing defined in two separate place.